### PR TITLE
clarifying that for the common case of a partitioned index on a parti…

### PR DESCRIPTION
**Pull request by davidbaxterbrowne**
…tioned Clustered Index, the partition column is already an index key column

### CODE REVIEW

**Comment by PRMerger9** [comment]
@davidbaxterbrowne : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/relational-databases/partitions/partitioned-tables-and-indexes.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ An index that is built on the same partition scheme as its corresponding table.
...
 
 #### Partitioning NonClustered Indexes
-When partitioning a unique nonclustered index, the index key must contain the partitioning column. When partitioning a nonunique, nonclustered index, [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] adds the partitioning column by default as a nonkey (included) column of the index to make sure the index is aligned with the base table. [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] does not add the partitioning column to the index if it is already present in the index. 
+When partitioning a unique nonclustered index, the index key must contain the partitioning column. When partitioning a nonunique, nonclustered index on, [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] adds the partitioning column by default as a nonkey (included) column of the index to make sure the index is aligned with the base table. [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] does not add the partitioning column to the index if it is already present in the index.  Nonunique, nonclustered indexes on a table with a clustered index always include the clustered index key columns as nonclustered index key columns.
```
Comment (davidbaxterbrowne) [inline docs/relational-databases/partitions/partitioned-tables-and-indexes.md:66]: the addition of "on" in the first line is a typo

Comment (pmasl) [reply docs/relational-databases/partitions/partitioned-tables-and-indexes.md:66]: While I understand why you'd want to add that generic info here, that info may be misunderstood as a characteristic of partitioned nonclustered indexes, and it's not. I'll add a note like "for more information on nonclustered indexes and index architecture, see https://docs.microsoft.com/en-us/sql/relational-databases/sql-server-index-design-guide?view=sql-server-ver15#Nonclustered"

**Review by pmasl — CHANGES_REQUESTED** [review]

File: docs/relational-databases/partitions/partitioned-tables-and-indexes.md (hunk 1 above)
Comment (julieMSFT) [reply docs/relational-databases/partitions/partitioned-tables-and-indexes.md:66]: @davidbaxterbrowne Hi David, The link to nonclustered indexes and index architecture has been made to the documentation. Thank you for your contribution to Microsoft Docs.

**Comment by julieMSFT** [comment]
- [ ] https://github.com/MicrosoftDocs/sql-docs-pr/pull/18249/files

**Comment by julieMSFT** [comment]
#please-close

**Comment by julieMSFT** [comment]
Update made in separate PR, 

- [ ] https://github.com/MicrosoftDocs/sql-docs-pr/pull/18249/files

**Comment by dbrownems** [comment]
But that information is not documented on the page you refer to.  It says that the clustered index key columns are included in the non-clustered index, but it does not clarify that they are stored as key columns for non-unique indexes and as row values for unique non-clustered indexes.  It's not really relevant in that context to clarify.  

But in this context it determines whether or not the partitioning column is added to the index, and clarifies that for all partitioned nonclustered indexes on clustered index tables, the partition key is already a non-clustered index key column.  For unique non-clustered indexes it's a requirement, and for non-unique non-clustered indexes the partitioning column is already there because the partitioning column is always a clustered index key column.

And so partitioning a non-clustered index on a clustered index table never adds any additional size to the index.

**Pull request closed without merging**